index.html

```diff
@@ -4800,16 +4800,15 @@ <h2>Contexts, Types, and Credential Schemas</h2>
         <h3>Base Context</h3>
           <p>
 The base context, located at
-<code>https://www.w3.org/2018/credentials/v1</code> with a SHA-256 digest of
-<strong><code>ab4ddd9a531758807a79a5b450510d61ae8d147eab966cc9a200c07095b0cdcc</code></strong>,
+<code>https://www.w3.org/ns/credentials/v2</code> with a SHA-256 digest of
+<strong><code>944167aaabd904ea9e35c98fd7e8794eb6dd42ae4666b036b171e87fc34cc7cc</code></strong>,
```

Comment:
Do we also (alternatively?) want a checksum that runs the context through JCS to avoid any whitespace issues?

Reply:
My recollection is that we had considered this in v1.0 and decided not to do that because 1) we wanted the JSON-LD Context that was fetched to be readable by developers, and 2) were concerned that people would just fetch that context, checksum it (without doing JCS), and then complain that the hashes didn't match (it would create a support burden). 

Perhaps we can provide a command that developers can use to easily verify the SHA256 hash? It's true that we could then integrate a JCS command to that command, but then, I don't know if there are any good JCS processors out there at present that would be broadly available to do this. What do you think of just adding a command so that they can verify the hash from the live published version?

Reply:
Attempted to partially address here: https://github.com/w3c/vc-data-model/pull/1003/commits/9b8c61e7907ada54d43233b494714723f0a660bf